docs/admin/customize.rst

```diff
@@ -70,41 +68,41 @@ Overall your module structure should look like:
         ├── addons.py
         └── checks.py
 
-You can find example application for custimizing Weblate at
-<https://github.com/WeblateOrg/customize-example>, it covers all topics
+You can find an example of for custimizing Weblate at
+<https://github.com/WeblateOrg/customize-example>, it covers all the topics
 described below.
 
-Changing logo
+Changing the logo
 -------------
 
-To change logo you need to create simple Django app which will contain static
-files which you want to overwrite (see :ref:`custom-module`). Then you add it
-into :setting:`django:INSTALLED_APPS`:
+1. Create a simple Django app containing the static files you want to overwrite
+(see :ref:`custom-module`).
+2. Add it to :setting:`django:INSTALLED_APPS`:
 
 .. code-block:: python
 
    INSTALLED_APPS = (
       # Add your customization as first
       'weblate_customization',
 
-      # Weblate apps are here...
+      # Weblate apps are here…
    )
 
-To adjust branding, you will most likely want to override following files:
+Branding appears in the following files:
 
 :file:`icons/weblate.svg`
     Logo shown in the navigation bar.
 :file:`logo-*.png`
-    Web icons depending on screen resolution and browser.
+    Web icons depending on screen resolution and web-browser.
 :file:`favicon.ico`
     Web icon used by legacy browsers.
 :file:`weblate-*.png`
-    In application avatars for bot or anonymous users. Also used by some browsers as a shortcut icon.
+    Avatars for bots or anonymous users. Some web-browsers use these as shortcut icons.
```

Comment:
What browsers?

Reply:
When you create a shortcut for a website on desktop (on computer) or in the launcher (on phone).